### PR TITLE
Update Troubleshooting Appium article

### PR DESCRIPTION
**Pull request by dino-vinkovic**
Jos malo updejta.
Prilagodio errore.

### CODE REVIEW

File: Automation/Mobile/Troubleshooting Appium.md [hunk 1]
```diff
@@ -20,36 +20,35 @@ NOTE:
...
-
-Appium runs tests alphabetically. It will first go through test files by name in alphabetical order. Then it will again go alphabetically through the tests inside each test file/class.
-
-One way to override that is to name your tests prefixed with numbers, something like:
-
-- test_001 (or test_001_001)
-- test_001_002 
-- ...
-
-If your tests are independent of each other, then you might not have to worry about the test order. In some cases, you might need to link a few tests together and in that case, you will have to keep this in mind.
```
Comment (matijadurdek) [inline Automation/Mobile/Troubleshooting Appium.md:38]: We don't need this section anymore in this article? 😄

**Review by matijadurdek — CHANGES_REQUESTED** [review]
Thanks for the update, Dino. Just the conflicts need to be resolved and we can merge the PR 😄

File: Automation/Mobile/Troubleshooting Appium.md (hunk 1 above)
Comment (dino-vinkovic) [reply Automation/Mobile/Troubleshooting Appium.md]: Discussed on Slack

**Review by matijadurdek — APPROVED** [review]
Thanks Dino. I'm merging the changes 😄

**Pull request merged**